### DISCUSSION TITLE
my tailwind css is not working for 4k screen

### DISCUSSION THREAD

**Discussion by parvez543** [tailwindlabs/tailwindcss#11203] [Help] [answered]
I have done some customize screen size in tailwind css which is not working at all

Here is the code

module.exports = {
    theme: {
      screens: {
        'sm': '576px',
        // => @media (min-width: 576px) { ... }
  
        'md': '960px',
        // => @media (min-width: 960px) { ... }
  
        'lg': '1440px',
        // => @media (min-width: 1440px) { ... }
  
        'xl': '1280px',
        // => @media (min-width: 1280px) { ... }
  
        '2xl': '1440px',
        // => @media (min-width: 1536px) { ... }
        '3xl': '1512px',
        '4xl': '1728px',
        '5xl': '1920px',
        '6xl': '2560px',
        '7xl': '3840px',
      }
    }
  }

here tailwind is working from sm-2xl and when i am going to use 3xl-7xl it is not working. 
![Screenshot from 2023-05-10 15-41-23](https://github.com/tailwindlabs/tailwindcss/assets/126660717/2d2256f6-be5c-4e0c-b6de-14d65a44f5f7)
![Screenshot from 2023-05-10 15-43-25](https://github.com/tailwindlabs/tailwindcss/assets/126660717/eeb11fe5-d963-4e11-9640-eb5fc791e619)

**Comment by wongjn** [comment, marked as answer]
You seem to have a `tailwind.config.js` but you are using the CDN.

If you would like to use `tailwind.config.js`, please [integrate Tailwind CSS as a Post CSS plugin](https://tailwindcss.com/docs/installation/using-postcss), use the [NPM module binary](https://tailwindcss.com/docs/installation) or use the [standalone executable](https://tailwindcss.com/blog/standalone-cli) to generate the CSS.

If you would like to use the CDN, modify the Tailwind configuration by modifying the `tailwind.config` in the global scope at runtime. See [step 2 of the CDN guide](https://tailwindcss.com/docs/installation/play-cdn) for an example.
